modules/misc/src/kernels/ADMatDiffusion.C

```diff
@@ -0,0 +1,30 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.htmlo
+#include "ADMatDiffusion.h"
+
+registerADMooseObject("MiscApp", ADMatDiffusion);
+
+defineADValidParams(ADMatDiffusion,
+                    ADDiffusion,
+                    params.addRequiredParam<MaterialPropertyName>(
+                        "diffusivity", "the name of the thermal conductivity material property"););
```

Comment:
Since the name of the parameter is `diffusivity` (which is nice and consistent with what we have in the rest of the framework), I think the doc string should be more like "The name of the diffusivity (or thermal conductivity, viscosity, etc. depending on the physics) material property"

Reply:
Whoops! I forgot to change the "thermal conductivity" to "diffusivity"